### ISSUE TITLE
Loss is nan

### ISSUE THREAD

**Issue by XuePengZhao** [twangnh/Distilling-Object-Detectors#4]
[session 1][epoch  1][iter 21000/23080] loss: nan, loss_sup: 8.9603, lr: 3.00e-03
                        fg/bg=(256/0), time cost: 19.056542
                        rpn_cls: nan, rpn_box: nan, rcnn_cls: nan, rcnn_box nan
[session 1][epoch  1][iter 21100/23080] loss: nan, loss_sup: 8.9857, lr: 3.00e-03
                        fg/bg=(256/0), time cost: 19.276745
                        rpn_cls: nan, rpn_box: nan, rcnn_cls: nan, rcnn_box nan
[session 1][epoch  1][iter 21200/23080] loss: nan, loss_sup: 8.6904, lr: 3.00e-03
                        fg/bg=(256/0), time cost: 19.519685
                        rpn_cls: nan, rpn_box: nan, rcnn_cls: nan, rcnn_box nan
[session 1][epoch  1][iter 21300/23080] loss: nan, loss_sup: 8.5541, lr: 3.00e-03
                        fg/bg=(256/0), time cost: 19.295986
                        rpn_cls: nan, rpn_box: nan, rcnn_cls: nan, rcnn_box nan
[session 1][epoch  1][iter 21400/23080] loss: nan, loss_sup: 8.6372, lr: 3.00e-03
                        fg/bg=(256/0), time cost: 19.508717

why the loss is nan？

**Comment by twangnh** [comment]
Hi @XuePengZhao have you solved the problem?

**Comment by raytrun** [comment]
When I trained the faster-rcnn with VGG11 and  lr  is 3.00e-03, I also got nan.  
Set  lr:=1.00e-04,  it will be ok.

**Comment by twangnh** [comment]
Sorry I didn't encounter the problem, is your vgg11 pretrained? actually I found lower lr cannot reproduce the result (eg., lr 2e-3 would get lower imitation result than 3e-3)
